### ISSUE TITLE
[Bug]: govTabs missing tabset ARIA role

### ISSUE THREAD

**Issue by cjrace** [dfe-analytical-services/shinyGovstyle#187]
### Overview

The govTabs function doesn't assign an ARIA role of tabset, so users relying on that aren't warned of the expected behaviour / highlighted to users when they saerch for tabs.

### Steps to Reproduce

Inspect the HTML of the showcase app.

### Expected Behavior

We need to assign `tablist` and `tab` roles within the govTabs function.

### Additional Information

User comments

“Although I found that the tabs element of the page worked as expected with JAWS, I noticed that the items which behaved like tabs, are identified as links by JAWS. Although this is not a significant problem for me as an advanced user of screen reading software, methods used to identify tabs can be different to those used to identify links. Including a clear indication to content and providing a consistent structure of navigation, will resolve this issue for users of screen reading software."

Page 23 of Sept 2025 accessibility audit report
